### DISCUSSION TITLE
Convective Outlook

### DISCUSSION THREAD

**Discussion by DinnerBeef** [weather-gov/api#627] [General]
Is there a way I can get the Convective Outlook images by the API or this there another spot I can get them

**Comment by akrherz** [comment]
If you just want the Storm Prediction Center images, the website provides these with predictable/stable URIs.  For example:

https://www.spc.noaa.gov/products/outlook/archive/2023/day1otlk_20230403_0100.html
https://www.spc.noaa.gov/products/outlook/archive/2023/day1otlk_20230403_0100_prt.gif
https://www.spc.noaa.gov/products/outlook/archive/2023/day1probotlk_20230403_0100_hail_prt.gif
etc

Otherwise, perhaps elaborate more on exactly what you are looking for.

**Comment by DinnerBeef** [comment]
yea that works

**Comment by ephcoding** [comment]
a little late to the party but the SPC's [MapServer](https://mapservices.weather.noaa.gov/vector/rest/services/outlooks/SPC_wx_outlks/MapServer) provides various Convective Outlook imagery & geometry formats.  So, you can export and render with Esri's [ArcGIS JS SDK](https://developers.arcgis.com/javascript/latest/) or simple GeoJSON that can be rendered with something like [D3](https://d3js.org/).
